mne/viz/backends/renderer.py

```diff
@@ -169,18 +193,18 @@ def use_3d_backend(backend_name):
 
     Parameters
     ----------
-    backend_name : {'mayavi', 'pyvista', 'notebook'}
+    backend_name : {'mayavi', 'pyvistaqt', 'notebook'}
         The 3d backend to use in the context.
     """
-    old_backend = _get_3d_backend()
-    set_3d_backend(backend_name)
```

Comment:
This behavior was wacky -- `_get_3d_backend` would set a backend (otherwise raise an error), only for it potentially to be un-set immediately by the following `set_3d_backend` call. The only reason I could see for this was that the old backend name was needed, so I just made `set_3d_backend` return the previous backend's name...